handler.py

```diff
@@ -219,8 +232,7 @@ def get_data(event, context):
             ))
         )
     start_index = None
-    if (event["queryStringParameters"] and
-            "index" in event["queryStringParameters"]):
+    if "index" in event.get("queryStringParameters", {}):
```

Comment:
nitpick: don't really need the second arg here

Reply:
Ah, right, because I was going to collapse that down before I spotted the log statement. Thanks!